src/SavingsDai.sol

```diff
@@ -235,6 +235,7 @@ contract SavingsDai {
         }
 
         emit Deposit(msg.sender, receiver, assets, shares);
+        emit Transfer(address(0), receiver, shares);
```

Comment:
Why is order different here than in _burn?

Reply:
Because it's the inverted order of deposit/withdraw. Deposit then Mint. Burn then Withdraw. It doesn't matter too much, but I feel that order makes more sense.

Reply:
I don't think so, all good